cypress/e2e/authenticator/Authenticator.spec.ts

```diff
@@ -11,17 +11,32 @@ describe("Authenticator", () => {
     cy.get("[data-cy=authTitle]").should("contain.text", "TIS Self-Service");
   });
 
+  it("Body should have the support FAQ and mailto links", () => {
+    cy.get("[data-cy=signInSupportLinks]")
+      .find("a")
+      .should($anchors => {
+        expect($anchors).to.have.length(2);
+        expect($anchors.first()).to.contain("FAQ");
+        expect($anchors.last()).to.contain("email");
+
+        const hrefs = $anchors.map<string>((_i, a) => {
```

Comment:
The ordering of JQuery .map callback params always confuses me! (I always think it will break if the index isn't the second param 😁 )

Reply:
It's a good shot. I have got rid of the .map(). It doesn't really need to map as there is only two elements.